packages/salesforcedx-vscode-core/test/vscode-integration/commands/util/libraryDeployResultParser.test.ts

```diff
@@ -151,102 +184,50 @@ describe('Deploy Parser', () => {
     status: ToolingDeployStatus.Error,
     components: [
       {
-        component: apexComponent,
+        component: apexComponentOne,
         status: ComponentStatus.Failed,
         diagnostics: [
           {
             type: 'Error',
-            filePath: apexClassPath,
+            filePath: apexClassPathOne,
             message: 'Unexpected error happened during deploy'
           }
         ]
       }
     ]
   };
 
-  // it('should create array of success info for updated class', async () => {
```

Comment:
these comments seem to be leftover from a previous merge, not sure if we kept them for something

Reply:
I'm not sure we've replaced these tests with new one and some of them seem valid

Reply:
I think these tests were replaced by the ones currently present- they test a level higher than `buildSuccesses` and `buildErrors` through the `resultParser` method